sdk/identity/identity-broker/README.md

```diff
@@ -0,0 +1,102 @@
+# Azure Identity Plugin for Brokered Authentication
+
+This package provides a plugin to the Azure Identity library for JavaScript ([`@azure/identity`](https://npmjs.com/package/@azure/identity)) that enables brokered authentication.
```

Comment:
`that enables using an authentication broker such as WAM.`

May want to link to https://learn.microsoft.com/en-us/azure/active-directory/develop/scenario-desktop-acquire-token-wam

Reply:
done